### PR TITLE
Updated rubygem-smart_proxy_discovery to 1.0.3 (DEB)

### PR DESCRIPTION
**Pull request by lzap**
1.10 as well please.

### CODE REVIEW

File: plugins/smart_proxy_discovery/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-smart-proxy-discovery (1.0.3) stable; urgency=low
```
Comment (mmoll) [inline plugins/smart_proxy_discovery/changelog]: `1.0.3-1` please

Comment (domcleal) [inline plugins/smart_proxy_discovery/changelog]: I don't know if @lzap used it, but I think the update script I contributed the other week leaves off the `-1`.  I've always been a little unclear about whether to include it or not for the first package of a new upstream version, but on re-reading [this bit of the Debian manual](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Version) (actually describing the binary control file), it's convention to include it.

Comment (domcleal) [inline plugins/smart_proxy_discovery/changelog]: #867 fixes this, but @lzap you still need to update this PR.

**Comment by domcleal** [comment]
Fixed and merged in 020dd29, thanks @lzap.

**Comment by domcleal** [comment]
Merged as 7aadef6 to deb/1.10.

**Pull request closed without merging**